services/avve_asset_apy/aave_apy.py

```diff
@@ -1,35 +1,75 @@
 from gql import gql, Client
 from gql.transport.requests import RequestsHTTPTransport
 
-from utilities import cruize_constants
 from utilities.enums import AssetCodes
 
 
-class AaveApy:
+class AaveApy(object):
+    def __init__(self):
+        self.aave_sub_graph_url = (
+            "https://api.thegraph.com/subgraphs/name/aave/protocol-v2"
+        )
+        self.avve_sub_graph_query = """
+                query {
+                  reserves (where: {
+                    usageAsCollateralEnabled: true
+                  }) {
+                    id
+                    name
+                    price {
+                      id
+                      priceInEth
+                    }
+                    liquidityRate
+                    variableBorrowRate
+                    stableBorrowRate
+                    aEmissionPerSecond
+                    vEmissionPerSecond
+                    decimals
+                    totalATokenSupply
+                    totalCurrentVariableDebt
+                    symbol
+                  }
+                }
+                """
+        self.aave_ray = 10.0**27
+        self.aave_asset_symbols = {"bitcoin": "WBTC", "ethereum": "WETH"}
+
     def fetch_asset_apys(self):
-        query = gql(cruize_constants.AVVE_SUB_GRAPH_QUERY)
+        query = gql(self.avve_sub_graph_query)
         transport = RequestsHTTPTransport(
-            url=cruize_constants.AAVE_SUB_GRAPH_URL,
+            url=self.aave_sub_graph_url,
             verify=True,
             retries=3,
         )
         client = Client(transport=transport)
 
         response = client.execute(query)
         apy_graph_data = response["reserves"]
-        subset_of_apy_asset = set(cruize_constants.AAVE_APY_ASSET)
+        subset_of_apy_asset = set(self.aave_asset_symbols.values())
         apy_data = [i for i in apy_graph_data if i["symbol"] in subset_of_apy_asset]
         asset_apy = {}
 
         for interval in range(len(apy_data)):
             token_liquidityRate = int(apy_data[interval]["liquidityRate"]) * 100
             asset_apy[AssetCodes.asset_name.value[apy_data[interval]["symbol"]]] = (
-                token_liquidityRate / cruize_constants.AAVE_RAY
+                token_liquidityRate / self.aave_ray
             )
 
         return asset_apy
 
+    def store_asset_apys(self, asset_apys=None):
+        from components import FirebaseDataManager
+
+        firebase_data_manager_obj = FirebaseDataManager()
```

Comment:
Had to add it here to prevent cyclic import error. Apparently components does not fully initialize so if we add these 2 lines at the top of the file, they result in a cyclic import error. -- we should fix this somehow